test/integration/helpers/shared_examples.rb

```diff
@@ -17,7 +17,7 @@
     it { should be_running }
   end
 end
-describe command('rpcinfo -p localhost') do
+describe bash('rpcinfo -p localhost') do
```

Comment:
I'd prefer to keep this using the ``command`` resource unless we absolutely need to run something using the bash interpreter.